TitleCreator.py

```diff
@@ -5,11 +5,12 @@
 
 def ofprefix():
     l = ['Viceroy', 'Commandant', 'Grand Poo-Bah', 'Archon', 'Duke', 'Chancellor', 'President', 'Marquis',
-            'Earl', 'Director', 'Chair', 'Head', 'Senior Director', 'Vice President', 'Dark Lord']
+            'Earl', 'Director', 'Chair', 'Head', 'Senior Director', 'Vice President', 'Dark Lord',
```

Comment:
Can we remove 'Administrator' due to https://github.com/gabinante/TitleCreator/issues/5 ? Or feel free to make a PR addressing this issue :)

Reply:
Done.